backend/de.metas.acct.base/src/main/java/de/metas/acct/impexp/AccountImportProcess.java

```diff
@@ -318,4 +319,31 @@ private static final Integer extractTreeIdOrNull(final Map<String, Object> map)
 
 		return treeId;
 	}
+	
+	private Set<Integer> retrieveImporteElements()
+	{
+		final IQueryBL queryBL = Services.get(IQueryBL.class);
+		return queryBL.createQueryBuilder(I_I_ElementValue.class)
+				.addOnlyActiveRecordsFilter()
+				.addEqualsFilter(I_I_ElementValue.COLUMNNAME_Processed, true)
+				.addOnlyActiveRecordsFilter()
+				.andCollect(I_I_ElementValue.COLUMN_C_Element_ID)
```

Comment:
what does this and Collect do?
feel free to explain verbally or with some pseudo-SQL..